react-native-reanimated.d.ts

```diff
@@ -187,10 +188,10 @@ declare module 'react-native-reanimated' {
           ? AnimateStyle<S[K]>
           :   
               // allow `number` where `string` normally is to support colors    
-                S[K] extends (string | undefined) ? S[K] | number : S[K]
+                S[K] extends (ColorValue | string | undefined) ? S[K] | number : S[K]
```

Comment:
I love it. the | undefined can be removed: (ColorValue | Angle) (Angle = string)

Reply:
please forget this comment, current typing is great.